tinywallet/tinywallet/screens.py

```diff
@@ -986,8 +971,9 @@ def loadClicked(self):
         if fd.exec_():
             fileNames = fd.selectedFiles()
             if len(fileNames) != 1:
-                log.error("More than 1 file selected for importing")
-                raise Exception("More than 1 file selected for importing")
+                msg = "more than one file selected for importing"
+                log.error(msg)
+                raise Exception(msg)
         else:
             raise Exception("no file selected")
```

Comment:
Are these exceptions necessary? Currently, if one cancels the file dialog, the app exits, which is not a good user experience.

Reply:
I agree. It should just close the dialog and wait. Should be a separate issue though.

Reply:
Made #172 about it.